src/platform/backends/qemu/qemu_virtual_machine.cpp

```diff
@@ -213,8 +238,6 @@ mp::QemuVirtualMachine::~QemuVirtualMachine()
             shutdown();
         }
     }
-
-    qemu_platform->remove_resources_for(vm_name);
```

Comment:
We still need this on purge, don't we?

Reply:
Not here.  Purge in the daemon already calls this directly.